bin/strata-client/src/extractor.rs

```diff
@@ -619,7 +621,7 @@ mod tests {
             "some deposits should have been randomly dispatched"
         );
 
-        let needle_index = rand::thread_rng().gen_range(0..dispatched_deposits.len());
+        let needle_index = rand::rngs::OsRng.gen_range(0..dispatched_deposits.len());
```

Comment:
We should move this to an import
```suggestion
        let needle_index = OsRng.gen_range(0..dispatched_deposits.len());
```